# Use EB modules for zlib and ncurses
# Author: Adam Huffman
# The Francis Crick Institute
diff -u samtools-1.3.1/Makefile samtools-1.3.1.new/Makefile
--- samtools-1.3.1/Makefile	2016-04-22 10:51:16.000000000 +0100
+++ samtools-1.3.1.new/Makefile	2016-04-23 23:53:26.266962157 +0100
@@ -82,13 +82,13 @@
 # TODO Use configure or htslib.pc to add -rdynamic/-ldl conditionally
 ALL_CPPFLAGS = -I. $(HTSLIB_CPPFLAGS) $(CPPFLAGS)
 ALL_LDFLAGS  = -rdynamic $(HTSLIB_LDFLAGS) $(LDFLAGS)
-ALL_LIBS     = -lz -ldl $(LIBS)
+ALL_LIBS     = -L$(EBROOTZLIB)/lib -lz -ldl $(LIBS)
 
 # Usually config.mk and config.h are generated by running configure
 # or config.status, but if those aren't used create defaults here.
 
 config.mk:
-	@sed -e '/^prefix/,/^LIBS/d;s/@Hsource@//;s/@Hinstall@/#/;s#@HTSDIR@#htslib-1.3.1#g;s/@HTSLIB_CPPFLAGS@/-I$$(HTSDIR)/g;s/@CURSES_LIB@/-lcurses/g' config.mk.in > $@
+	@sed -e '/^prefix/,/^LIBS/d;s/@Hsource@//;s/@Hinstall@/#/;s#@HTSDIR@#htslib-1.3.1#g;s/@HTSLIB_CPPFLAGS@/-I$$(HTSDIR)/g;s/@CURSES_LIB@/-L$$(EBROOTNCURSES)\/lib -lcurses/g' config.mk.in > $@
 
 config.h:
 	echo '/* Basic config.h generated by Makefile */' > $@
